Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/snowbridge-dashboards.md

---
id: snowbridge-dashboards
title: Snowbridge on BridgeHub
sidebar_label: Snowbridge

> description: Snowbridge is a general-purpose, trustless, and decentralized bridge that connects
>   Polkadot and Ethereum. This bridge facilitates communication between any Polkadot parachain and
>   Ethereum using Polkadot's XCMP messaging protocol. Parachains can send XCM instructions to
>   BridgeHub, leveraging Snowbridge for cross-chain interactions.
> 
>   For more details on bridging with Kusama, please refer to our [BridgeHub
>   Dashboard](https://dune.com/substrate/bridgehub).
> 

Reviewer: ```suggestion
description: Snowbridge is a general-purpose, trustless, and decentralized bridge that connects
  Polkadot and Ethereum.
```

keywords: [polkadot, ethereum, dashboard, dune, snowbridge]
slug: ../snowbridge-dashboards
---

# Snowbridge Dashboards

## Overview

Snowbridge is a general-purpose, trustless, and decentralized bridge that connects Polkadot and
Ethereum. This bridge facilitates communication between any Polkadot parachain and Ethereum using
Polkadot's XCMP messaging protocol. Parachains can send XCM instructions to BridgeHub, leveraging
Snowbridge for cross-chain interactions.

For more details on bridging with Kusama, please refer to our
[BridgeHub Dashboard](https://dune.com/substrate/bridgehub).

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Snowbridge protocol:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Snowbridge protocol:
```

- [Snowbridge](https://dune.com/substrate/snowbridge): A comprehensive analysis of Snowbridge,
  including: Ethereum Related On Chain Data Analysis, Analysis of Messages with Ethereum.

## Key Tables

Data from the Snowbridge protocol is organized into several key tables:

- `bridgehub.balances`
- `bridgehub.blocks`
- `bridgehub.calls`
- `bridgehub.events`,
- `bridgehub.extrinsics`
- `bridgehub.transfers`

## Useful Queries

Currently, there are no specific useful queries provided. Please check back later as this section

> will be updated with materialized queries for Snowbridge.

Reviewer: ```suggestion
Currently, there are no specific queries provided.
```
